openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -171,14 +183,22 @@ class CreateUpdatePermissionsMixin(UserMixin, UserScopeMixin, Action):
             "vote_delegated_to_id",
             "vote_delegations_from_ids",
             "is_present_in_meeting_ids",
+            "is_present",  # participant import
         ],
         "C": ["meeting_id", "group_ids"],
         "D": ["committee_ids", "committee_management_ids"],
-        "E": ["organization_management_level", "saml_id"],
+        "E": ["organization_management_level"],
         "F": ["default_password"],
         "G": ["is_demo_user"],
+        "H": ["saml_id"],
     }
 
+    def perform(
+        self, action_data: ActionData, user_id: int, internal: bool = False
+    ) -> Tuple[Optional[WriteRequest], Optional[ActionResults]]:
+        self.internal = internal
+        return super().perform(action_data, user_id, internal)
```

Comment:
Ah, I had a similar change in https://github.com/OpenSlides/openslides-backend/pull/2022. I prefer the global approach of solving this in the `action.py` instead of each action implementing their own solution.

Reply:
done